docs/firebase_layers.js

```diff
@@ -36,7 +39,8 @@ function createStyleFunction(colorFunctionProperties) {
   const field = colorFunctionProperties['field'];
   switch (colorFunctionProperties.currentStyle) {
     case ColorStyle.SINGLE:
-      const color = colorMap.get(colorFunctionProperties.color);
+      // Drop opacity for single-color.
```

Comment:
why?

Reply:
Just preserving behavior: we didn't specify opacity before, which made it solid.

Reply:
If it's not too much work for you, I'd rather update single-color to use opacity. Will let us drop this special check and standardize across coloring schemes. If that's going to require a more extensive PR though happy to approve as is.

Reply:
Just changing the special-casing is easy (until Travis tells me otherwise), done.